content/en/workshop/index.adoc

```diff
@@ -86,7 +87,7 @@ Besides long paper submissions, we also invite previously published or ongoing a
 
 == Awards
 
-A best paper award will be presented at the workshop and will be announced on our website.
+The best paper award will be presented at the workshop and will be announced on our website.
```

Comment:
"The" seems to refer to an award that was previously mentioned (native speakers correct me if I'm wrong, please! :grimacing:) so I'd be in favor of "A" here.

Reply:
Either way reads more or less reasonable. Changing